qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -49,9 +49,13 @@
 logger = logging.getLogger(__name__)
 
 
-def decompose_two_qubit_product_gate(special_unitary_matrix):
-    """Decompose U = Ul⊗Ur where U in SU(4), and Ul, Ur in SU(2).
-    Throws QiskitError if this isn't possible.
+def decompose_two_qubit_product_gate(special_unitary_matrix: np.ndarray):
+    r"""Decompose :math:`U = U_l \otimes U_r` where :math:`U` is in :math:`SU(4)`,
+    and :math:`U_l`, :math:`U_r` are in :math:`SU(2)`.
+    Args:
```

Comment:
This could be abbreviated if you want (very optional suggestion 😄)
```suggestion
    r"""Decompose :math:`U = U_l \otimes U_r` where :math:`U \in SU(4)`,
    and :math:`U_l,~U_r \in SU(2)`.

    Args:
```